src/components/organization/create-dho-widget.vue

```diff
@@ -4,11 +4,12 @@ export default {
   components: {
     Widget: () => import('~/components/common/widget.vue')
   }
+
 }
 </script>
 
 <template lang="pug">
-widget.q-my-md.position-relative(title="Create your DHO")
-  .text-ellipsis.text-body.q-my-xl Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-  q-btn.q-mt-xl.q-px-lg(rounded color="primary" no-caps @click="$router.push({ name: 'dao-launcher' })") New DHO
+widget.q-my-md.position-relative(title="Create new DAO")
+  //- .text-ellipsis.text-body.leading-loose.q-my-md Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
```

Comment:
@arsenijesavic do not leave commented out code. Either write it out as TODO if it is temporary or better yet. Delete it. You can always find it in the git history